src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -473,18 +483,21 @@ def bad_move(self, order: str) -> bool:
 
     def update_allies_and_foes(self) -> None:
         power_stance = self.stance.stance[self.power_name]
+        neighbors = neighboring_opps(self.game, self.power_name, self.opponents)
 
         self.allies = [
             pow for pow in self.opponents if power_stance[pow] > self.ally_threshold
         ]
         self.foes = [
             pow for pow in self.opponents if power_stance[pow] <= self.enemy_threshold
         ]
+
+        # disable aggressive move filtering against allies if there are no neighboring enemies
+        self.is_filtering_moves = any(pow in neighbors for pow in self.foes)
```

Comment:
Let's add a call to `send_intent_log()` here because this is worth recording. It should say something similar to the comment.

Reply:
fixed